server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -777,6 +777,7 @@ func (e *EventChannel) FinalizeInvocation(iid string) error {
 		invocation,
 		map[string]*url.URL{
 			"timing_profile/" + e.beValues.ProfileName(): e.beValues.ProfileURI(),
+			"test_log/" + e.beValues.TestLogName():       e.beValues.TestLogURI(),
```

Comment:
we'll probably want to scan the BES stream here to get a list of test logs. In `webhookNotifier` (which also runs in the background, similarly to statsRecorder), we are already looking up the full invocation BES event stream. Maybe we could somehow reuse that work? ideally in a way where we aren't keeping the full BES stream buffered in memory for too long.

Reply:
This seemed very complicated to do; I think it is work worth doing, but may be outside the scope of this PR. We probably want to make some sort of BuildEventStreamPostProcess pipeline that we can feed arbitrary functions into and then it can load the BES (ideally in small chunks via streaming) and map all the functions over the events?